### PR TITLE
Update bounding volume semantics based on feedback 

### PR DESCRIPTION
**Pull request by ptrgags**
in #9549, @lilleyse gave me some additional feedback after the PR was merged, see here: https://github.com/CesiumGS/cesium/pull/9549#pullrequestreview-667269732

This small PR addresses those two minor details.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ptrgags!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/bounding-volume-semantic-updates/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Source/Scene/MetadataSemantic.js [hunk 1]
```diff
@@ -57,6 +57,86 @@ var MetadataSemantic = {
...
+   */
+  CONTENT_BOUNDING_SPHERE: "CONTENT_BOUNDING_SPHERE",
+  /**
+   * The minimum height of the content of a tile above (or below) the WGS84 ellipsoid, stored as a <code>FLOAT32</code> or a <code>FLOAT64</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  CONTENT_MINIMUM_HEIGHT: "CONTENT_MINIMUM_HEIGHT",
```
Comment (ptrgags) [inline Source/Scene/MetadataSemantic.js:131]: Content min/max height are in the spec but not used in the implementation. Should we include them here or no?

File: Source/Scene/MetadataSemantic.js [hunk 2]
```diff
@@ -57,6 +57,86 @@ var MetadataSemantic = {
    * @private
```
Comment (ptrgags) [inline Source/Scene/MetadataSemantic.js:57]: (Above the fold) - it looks like we had constants for `MINIMUM_HEIGHT` and `MAXIMUM_HEIGHT`, which are not in the current [Semantics spec](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata/Semantics). 

Should we remove these two older constants or does something need them?

Comment (lilleyse) [reply Source/Scene/MetadataSemantic.js:57]: Yes you can remove anything that's outdated

File: Source/Scene/MetadataSemantic.js [hunk 3]
```diff
@@ -57,6 +57,86 @@ var MetadataSemantic = {
...
+  /**
+   * A bounding region for a tile, stored as an array of 6 <code>FLOAT32</code> or <code>FLOAT64</code>. The components are <code>[west, south, east, north, minimumHeight, maximumHeight]</code>. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  TILE_BOUNDING_REGION: "TILE_BOUNDING_REGION",
+  /**
+   * A bounding sphere for a tile, stored as an array of 6 <code>FLOAT32</code> or <code>FLOAT64</code>. The components are <code>[centerX, centerY, centerZ, radius]</code>. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
```
Comment (lilleyse) [inline Source/Scene/MetadataSemantic.js:77]:
```suggestion
   * A bounding sphere for a tile, stored as an array of 4 <code>FLOAT32</code> or <code>FLOAT64</code> components. The components are <code>[centerX, centerY, centerZ, radius]</code>. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
```

Note both changes above.

File: Source/Scene/MetadataSemantic.js [hunk 4]
```diff
@@ -57,6 +57,86 @@ var MetadataSemantic = {
    * @private
    */
   ID: "ID",
+  /**
+   * A bounding box for a tile, stored as an array of 12 <code>FLOAT32</code> or <code>FLOAT64</code>. The components are the same format as for <code>boundingVolume.box</code> in 3D Tiles 1.0. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
```
Comment (lilleyse) [inline Source/Scene/MetadataSemantic.js:61]:
```suggestion
   * A bounding box for a tile, stored as an array of 12 <code>FLOAT32</code> or <code>FLOAT64</code> components. The components are the same format as for <code>boundingVolume.box</code> in 3D Tiles 1.0. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
```

File: Source/Scene/MetadataSemantic.js [hunk 5]
```diff
@@ -57,6 +57,86 @@ var MetadataSemantic = {
...
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  TILE_BOUNDING_BOX: "TILE_BOUNDING_BOX",
+  /**
+   * A bounding region for a tile, stored as an array of 6 <code>FLOAT32</code> or <code>FLOAT64</code>. The components are <code>[west, south, east, north, minimumHeight, maximumHeight]</code>. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
```
Comment (lilleyse) [inline Source/Scene/MetadataSemantic.js:69]:
```suggestion
   * A bounding region for a tile, stored as an array of 6 <code>FLOAT64</code> components. The components are <code>[west, south, east, north, minimumHeight, maximumHeight]</code>. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
```

Region components must be float64 for precision reasons:
https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata/Semantics#tile_bounding_region

File: Source/Scene/MetadataSemantic.js [hunk 6]
```diff
@@ -57,6 +57,86 @@ var MetadataSemantic = {
...
+  /**
+   * The minimum height of a tile above (or below) the WGS84 ellipsoid, stored as a <code>FLOAT32</code> or a <code>FLOAT64</code>. This semantic is used to tighten bounding regions implicitly calculated <code>3DTILES_implicit_tiling</code>.
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  TILE_MINIMUM_HEIGHT: "TILE_MINIMUM_HEIGHT",
+  /**
+   * The maximum height of a tile above (or below) the WGS84 ellipsoid, stored as a <code>FLOAT32</code> or a <code>FLOAT64</code>. This semantic is used to tighten bounding regions implicitly calculated <code>3DTILES_implicit_tiling</code>.
```
Comment (lilleyse) [inline Source/Scene/MetadataSemantic.js:93]:
```suggestion
   * The maximum height of a tile above (or below) the WGS84 ellipsoid, stored as a <code>FLOAT32</code> or a <code>FLOAT64</code>. This semantic is used to tighten bounding regions implicitly calculated in <code>3DTILES_implicit_tiling</code>.
```

**Comment by ptrgags** [comment]
@lilleyse updated!

**Comment by lilleyse** [comment]
Thanks @ptrgags !

**Pull request merged**